studio/pages/project/[ref]/reports/database.tsx

```diff
@@ -285,6 +133,55 @@ const DatabaseUsage = observer(() => {
               </div>
             </Panel.Content>
           </Panel>
+
+          <Panel
+            title={
+              <div className="flex justify-between w-full">
+                <h2>Database size</h2>
+                <span className="text-lg tracking-wide">{formatBytes(databaseSizeBytes)}</span>
+              </div>
+            }
+          >
+            <Panel.Content>
+              <div className="space-y-1">
+                <span className="text-md">Largest Objects</span>
+
+                <Loading active={!loadedLargestObjects}>
+                  <div className="space-y-3 mt-4">
+                    {largestObjects?.result?.map((object) => (
+                      <div key={`${object.schema_name}.${object.relname}`}>
+                        <SparkBar
```

Comment:
we can display this in a 2 column table.  Don't need to show 10 items as not all will be interesting and will take up unnecessary vertical height. Showing top 3 with an expand for 10 will be enough. Can also link to the scripts query page to make it more usable, like how the API reports is styled [here](https://github.com/supabase/supabase/pull/17739) in this PR.

Reply:
we can remove use of the sparkbar and instead show it as a % in brackets next to the value. In this situation it would be better to show a proportion visualization, such as segmented horizontal bar chart.

Reply:
![Screenshot 2023-10-09 at 15 40 41](https://github.com/supabase/supabase/assets/14073399/f9a68c3a-c806-4797-af3b-20a13f0fd5e6)

Was inspired by Vercel for the current styling (even though Vercel puts the text in the bar)

Reply:
Do we have a proportion visualization anywhere already?

Reply:
Can just add the percentage in brackets for now